lab_3_ann_retriever/main.py

```diff
@@ -253,9 +376,24 @@ def index_documents(self, documents: list[str]) -> bool:
 
         In case of corrupt input arguments, False is returned.
         """
+        if not (isinstance(documents, list) and documents and all(
+                isinstance(doc, str) for doc in documents)):
+            return False
+        self._documents = documents
+
+        for document in documents:
+            indexed_document: tuple[float, ...] | None = self._index_document(document)
+            if indexed_document is not None:
+                self._document_vectors.append(indexed_document)
+            else:
+                return False
+
+        if None in self._document_vectors:
```

Comment:
return `None not in self._document_vectors`

Reply:
done